src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,210 @@
+---
```

Comment:
Not sure if this is just MDX - but seem to be missing the left-side navigation on this page

Reply:
Yes, it should be fine now I've rebased it with the MDX updates.